Added brief mention of custom_offset_string field (#390)
konfuzio-ai/konfuzio-sdk: docs/web/api-v3.md

  - `offset_string` (optional when user-provided): the text contained inside this Span. This can be manually set by the
    user if the text existing at the specified coordinates is wrong.
  - `offset_string_original` (read-only): the text that was originally present at the specified coordinates. This is

>     usually the same as `offset_string` unless it has been changed manually. If the values of those two fields are different, the field `custom_offset_string` will be set to `true`.

Reviewer: ```suggestion
    usually the same as `offset_string` unless it has been changed manually. If the value of `offset_string_original` differs from the value of `offset_string`, the field `custom_offset_string` will be set to `true`.
```
